src/RuleErrors/DisallowedConstantRuleErrors.php

```diff
@@ -39,7 +39,7 @@ public function get(string $constant, Scope $scope, ?string $displayName, array
 				$errorBuilder = RuleErrorBuilder::message(sprintf(
 					'Using %s%s is forbidden, %s',
 					$disallowedConstant->getConstant(),
-					$displayName && $displayName !== $disallowedConstant->getConstant() ? ' (as ' . $displayName . ')' : '',
+					$displayName && ltrim($displayName, '\\') !== $disallowedConstant->getConstant() ? ' (as ' . $displayName . ')' : '',
```

Comment:
ℹ️ `$displayName` will be "root" namespaced (`\Waldo\Quux\Blade::DECKARD`) while `$disallowedConstant->getConstant()` will not (`Waldo\Quux\Blade::DECKARD`)

Reply:
Just realized there's too many `ltrim`s already, even before this PR, I'll think of a better way 😊

Reply:
#187 deals with `ltrim`s